deploy/helm/sumologic/values.yaml

```diff
@@ -165,6 +165,24 @@ sumologic:
     ## Option to specify the Fluentd buffer as file/memory.
     buffer: "memory"
 
+    securityContext:
+      fsGroup: 999
+
+    ## Persist data to a persistent volume; Must be used together with Fluentd file buffer (fluentd.buffer="file")
```

Comment:
do we want to require them to set two values, or let this one take **precedence** over the other one? I could go either way, what are other charts doing?

Reply:
I think you are right, having this one takes precedence makes sense. I don't see any charts using two values to enable persistence